src/main/java/coffee/order/controller/Customer.java

```diff
@@ -0,0 +1,85 @@
+package coffee.order.controller;
+
+import coffee.order.domain.*;
+import coffee.order.view.InputView;
+import coffee.order.view.OutputView;
+
+import java.util.List;
+
+import static coffee.order.domain.Answer.YES;
+
+public class Customer {
+    public static final int COUNT_INDEX = 1;
+    public static final String NO_VALUE = "";
```

Comment:
```suggestion
    private static final int COUNT_INDEX = 1;
    private static final String NO_VALUE = "";
```
외부로 노출하지 않아도 되는 상수를 선언하는 경우 private 접근 제한자를 사용하는 것이 좋을 것 같습니다.